### PR TITLE
Update logo and copy

### PR DESCRIPTION
**Pull request by TaiWilkin**
## Overview

Updates the logo, powered-by footer image, newsletter signup copy, and facility status report copy. 

Connects #1432, #1438, #1439

## Demo

Screen Shot 2021-08-04 at 3 16 42 PM
Screen Shot 2021-08-04 at 3 19 40 PM
Screen Shot 2021-08-04 at 3 20 12 PM
Screen Shot 2021-08-04 at 3 20 49 PM

## Testing Instructions

* Run `./scripts/server`
* Navigate to [embedded map](http://localhost:6543/?embed=1&contributors=3) (replace the contributor with a contributor with an embed config) and view the footer image. 
* Navigate to the full website and view the logo in the header
* Navigate to a facility and click 'report facility as closed'. Check the text. 
* Log out and select 'register'. Check the newsletter signup text. 

## Checklist

- [x] `fixup!` commits have been squashed
- [x] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines

### CODE REVIEW

**Review by jwalgran — COMMENTED** [review]
I a couple issues, one of which is a 🤦 from me, recommending the PNG version of the logo rather than the SVG.

- The new main logo is appearing smaller. This may be resolved by using
[OpenApparelRegistry_logo_black.svg.zip](https://github.com/open-apparel-registry/open-apparel-registry/files/6939607/OpenApparelRegistry_logo_black.svg.zip) rather than the PNG. We should also use the SVG version of "powered by"
[OAR_PoweredBy_white.svg.zip](https://github.com/open-apparel-registry/open-apparel-registry/files/6939635/OAR_PoweredBy_white.svg.zip)

Screen Shot 2021-08-05 at 7 49 13 AM


- The mailing list checkbox is no longer on the same line as the text now that the text is longer.
Screen Shot 2021-08-05 at 7 51 01 AM

File: src/app/src/components/Navbar.jsx [hunk 1]
```diff
@@ -218,7 +218,11 @@ function Navbar({ openGDPR }) {
                             href="/"
                             style={{ display: 'inline-flex' }}
                         >
-                            <img src={logo} className="App-logo" alt="logo" />
+                            <img
+                                src={logo}
+                                style={{ height: '60px' }}
```
Comment (TaiWilkin) [inline src/app/src/components/Navbar.jsx:223]: The new image has white space at the top and bottom, causing a 40px high image to appear smaller. Increasing the size corrects this. (Adjusting the CSS didn't correct the issue, despite clearing the cache/etc., but using `styles` directly worked.)

File: src/app/src/components/ControlledCheckboxInput.jsx [hunk 2]
```diff
@@ -37,16 +41,21 @@ export default function ControlledCheckboxInput({
 
     return (
         <Fragment>
-            <MaterialCheckbox
-                id={id}
-                color="primary"
-                onChange={onChange}
-                style={controlledCheckboxInputStyles.checkbox}
-                checked={checked}
+            <FormControlLabel
```
Comment (TaiWilkin) [inline src/app/src/components/ControlledCheckboxInput.jsx:44]: The MaterialUI FormControlLabel wraps the label and checkbox component, preventing the two elements from splitting onto separate lines at small screen sizes.

**Review by jwalgran — APPROVED** [review]
👍

File: src/app/src/components/ReportFacilityStatus.jsx [hunk 3]
```diff
@@ -128,7 +128,12 @@ function ReportFacilityStatus({
                         className={classes.description}
                     >
                         Please provide information the OAR team can use to
-                        verify this status change.
+                        verify this status change. Should you need to share or
+                        attach a document or screenshot, please email it to
```
Comment (jwalgran) [inline src/app/src/components/ReportFacilityStatus.jsx:132]: We need to force a " " in here between "to" and the email address. Screen Shot 2021-08-06 at 9 00 30 AM

Comment (TaiWilkin) [reply src/app/src/components/ReportFacilityStatus.jsx]: Updated! Good catch 👍

**Pull request merged**